package.json

```diff
@@ -48,7 +48,7 @@
     "coveralls": "^2.11.4",
     "eslint": "^3.1.0",
     "istanbul": "^0.4.0",
-    "jscs": "^3.0.0",
+    "jscs": "^2.11.0",
```

Comment:
jscs@3.x was hanging for some reason on OSX, and I'm not inclined to spend time trying to figure out why, since it is deprecated already.